README.md

```diff
@@ -1,17 +1,134 @@
-# UCAN IPLD Schema
+# UCAN IPLD Schema v0.1.0
 
 ## Editors
 
-* _
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
 
 ## Authors
 
-* _
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
 # 0 Abstract
 
-## Language
+[Interplanetary Linked Data (IPLD)](https://ipld.io/) is a consistent, highly general data model for content addressed linked data forming any DAG. This data model provides a convenient pivot between many serializations of the same information. Being a chained token model, UCANs can be very naturally expressed via IPLD. However, JWTs are not fully deterministic due to whitespace and key ordering. This specification defines an IPLD Schema for UCAN, and a canonical JWT serialization for compatibility with all other clients.
+
+# 1 Motivation
+
+The [core UCAN specification](https://github.com/ucan-wg/spec) is defined as a JWT to make it easily adoptable with existing tools, and as a common format for all implementations. There are many cases where different encodings are preferred for reasons such as compactness, machine-efficient formats, and so on. This specification outlines a format based on IPLD that can be deterministically encoded and decoded between many serialization formats, while still being able to encode as JWT for compatibility.
+
+# 2 IPLD Schema
+
+Unlike a JWT, the IPLD encoding of UCAN does not require separate header, claims, and signature fields.
+
+```ipldsch
+type UCAN struct {
+  version String
+
+  issuer DID
+  audience DID
+  signature Signature
+
+  capabilities [Capability]
+  proofs [&UCAN]
+  expiration Int
+
+  facts [Fact]
+  nonce optional String
+  notBefore optional Int
+} representation map {
+  field facts default []
+  field proofs default []
+}
+```
+
+## 2.1 DID
+
+DIDs MUST be encoded as [DID](https://www.w3.org/TR/did-core/)s. The [`did:key` method](https://w3c-ccg.github.io/did-method-key/) is RECOMMENDED as it is self-contained.
+
+``` ipldsch
+type DID = String
```

Comment:
I have come to agree with the Ceramic folks about encoding this as a string instead of a key. We did the same thing in the Fission database initially (having assumed that we would only have `did:key`s), but have now had to migrate the DB to handle other DID methods as well. It basically can't hurt, even if you need to parse the key. The good news is that for `did:key`s, you can very easily put parsed keys in a memoization table.

Reply:
What I have suggested in the discussion is to use multiformats representation instead. In fact we could and probably should have `did` multiformat so we could represent more compactly.

That said I'm not going to die on this hill & if everyone are in favor of strings I'm ok with that